builder.py

```diff
@@ -149,14 +149,10 @@ def BuildQueueCleanup(self, queue):
         queue.append('date')
         # Basic steps are:
         #   1. git stash (in each subproject)
-        #   2. git clean -fdx (in each subproject)
-        #   3. git fetch (in each subproject)
+        #   2. git fetch (in each subproject)
         queue.append('git stash')
         queue.append('git submodule foreach git stash')
         #
-        queue.append('git clean -fdx')
-        queue.append('git submodule foreach git clean -fdx')
-        #
         queue.append('git fetch')
         queue.append('git submodule foreach git fetch')
```

Comment:
Okay, added --recurse-submodules, and removed line 161/157. Testing indicated that I did need the --recurse-submodules flag to do what I wanted.